tests/test_schism.py

```diff
@@ -112,8 +112,15 @@ def test_answer(tmpdir, case):
 
 def test_schism_meteo_split_by(tmpdir):
     model_description = copy.deepcopy(case1)
-    model_description.update({"rpath": tmpdir, "meteo_split_by": "hour", "update": ["meteo"]})
+    model_description.update(
+        {
+            "rpath": tmpdir,
+            "meteo_split_by": "1D",
+            "time_frame": "144H",
```

Comment:
In order to let the tests pass we need to increase `time_frame`, too. Regardless of the temporal "span" of the grib file, if time_frame is less than 24H then we will create a single sflux file anyhow.

I guess that this should be documented and/or we could have a check for this in the code.